File: iniciante/python/1008-salario.py
def salario(numero_de_funcionarios, horas_trabalhadas, valor_por_hora):
    salary = horas_trabalhadas * valor_por_hora
    saida = f'NUMBER = {numero_de_funcionarios}\n' \
            f'SALARY = U$ {salary:.2f}'
    return print(saida)


n_funcionarios = int(input())
hrs_trabalhadas = int(input())
valor_hora = float(input())
salario(n_funcionarios, hrs_trabalhadas, valor_hora)

File: iniciante/c/1008-salario.c
#include <stdio.h>

int main(){
    int n_funcionario, horas_trabalhas;
    float valor_por_hora, salario;
    scanf("%d %d %f", &n_funcionario, &horas_trabalhas, &valor_por_hora);
    salario = horas_trabalhas * valor_por_hora;
    printf("NUMBER = %d\nSALARY = U$ %.2f\n", n_funcionario, salario);
    return 0;
}
